lib/req/test.ex

```diff
@@ -154,6 +156,17 @@ defmodule Req.Test do
 
   require Logger
 
+  @typedoc """
+  A stub is an atom that scopes stubbed requests.
+
+  A common choice for this is the module name for the Req-based client that you
+  are using `Req.Test` for. For example, if you are using `MyApp.Weather` as
+  your client, you can use `MyApp.Weather` as the stub name in functions
+  like `stub/1` and `stub/2`.
+  """
+  @typedoc since: "0.4.15"
```

Comment:
It's fine to add it but fyi I'll probably remove it when I finally release v1.0.

Reply:
Yep makes sense to me 🙃